scripts/asciidoc2html.py

```diff
@@ -51,37 +59,35 @@ def __init__(self, asciidoc, website):
         self._tempdir = None
         self._failed = False
 
-    def prepare(self):
+    def prepare(self) -> None:
         """Get the asciidoc command and create the homedir to use."""
         self._cmd = self._get_asciidoc_cmd()
-        self._homedir = tempfile.mkdtemp()
-        self._themedir = os.path.join(
-            self._homedir, '.asciidoc', 'themes', 'qute')
-        self._tempdir = os.path.join(self._homedir, 'tmp')
-        os.makedirs(self._tempdir)
-        os.makedirs(self._themedir)
-
-    def cleanup(self):
+        self._homedir = pathlib.Path(tempfile.mkdtemp())
+        self._themedir = self._homedir / '.asciidoc' / 'themes' / 'qute'
+        self._tempdir = self._homedir / 'tmp'
+        self._tempdir.mkdir(parents=True)
+        self._themedir.mkdir(parents=True)
+
+    def cleanup(self) -> None:
         """Clean up the temporary home directory for asciidoc."""
         if self._homedir is not None and not self._failed:
             shutil.rmtree(self._homedir)
 
-    def build(self):
+    def build(self) -> None:
         """Build either the website or the docs."""
         if self._website:
             self._build_website()
         else:
             self._build_docs()
             self._copy_images()
 
-    def _build_docs(self):
+    def _build_docs(self) -> None:
         """Render .asciidoc files to .html sites."""
-        files = [('doc/{}.asciidoc'.format(f),
-                  'qutebrowser/html/doc/{}.html'.format(f))
+        files: List[Tuple[pathlib.Path, pathlib.Path]] = [(pathlib.Path('doc/{}.asciidoc'.format(f)),
```

Comment:
You'll need to use the comment syntax here to keep compatibility for Python 3.5.